### ISSUE TITLE
DEMO

### ISSUE THREAD

**Issue by standarddev** [kosukeimai/wru#16]
When I run the race.pred function with the demo = FALSE, the program runs fine.

When I run it with demo = TRUE, I get this error (I have an api, I just removed it)

dcounty <- race.pred(sample, name.clean = TRUE,census="county", census.key="...",demo=TRUE)
[1] "State  IL"
Error in `[<-.data.frame`(`*tmp*`, voters.census$sex == 0 & voters.census$agecat ==  : 
  missing values are not allowed in subscripted assignments of data frames

My dataset has age and sex as numeric variables. What is wrong?

**Comment by standarddev** [comment]
I believe it is because my age ranges from 16 and above. Is it true that the age must be greater than or equal to 18 in order to use?

**Comment by kkprinceton** [comment]
Yes, I believe so, as the function was originally intended for registered voters. Would you please try removing ages under 18 and rerunning the function? In the meantime, we will update the package to handle all ages without giving an error.

**Comment by standarddev** [comment]
blockquote, div.yahoo_quoted { margin-left: 0 !important; border-left:1px #715FFA solid !important; padding-left:1ex !important; background-color:white !important; } Wonderful! I will try and let you know how it goes. 
How long do you think it will take to update? A week? A month? 
I really appreciate the help. 
Thank you,Robin 

Sent from Yahoo Mail for iPhone


On Monday, February 27, 2017, 4:52 PM, kkprinceton <notifications@github.com> wrote:


Yes, I believe so, as the function was originally intended for registered voters. Would you please try removing ages under 18 and rerunning the function? In the meantime, we will update the package to handle all ages without giving an error.

**Comment by kkprinceton** [comment]
I will aim for the end of the week. I have edited the code, but we need to test it out.

**Comment by standarddev** [comment]
blockquote, div.yahoo_quoted { margin-left: 0 !important; border-left:1px #715FFA solid !important; padding-left:1ex !important; background-color:white !important; } Thank you so much!!



Sent from Yahoo Mail for iPhone


On Monday, February 27, 2017, 5:26 PM, kkprinceton <notifications@github.com> wrote:


I will aim for the end of the week. I have edited the code, but we need to test it out.

**Comment by standarddev** [comment]
blockquote, div.yahoo_quoted { margin-left: 0 !important; border-left:1px #715FFA solid !important; padding-left:1ex !important; background-color:white !important; } Hello,
Just wanted to update you that when I remove people under 18, it works correct. 
Thanks again for updating the code so that it'll be able to run all ages. 
Robin 


Sent from Yahoo Mail for iPhone


On Monday, February 27, 2017, 5:26 PM, Robin Parente <robin.parente@yahoo.com> wrote:

#yiv7677739100 blockquote, #yiv7677739100 div.yiv7677739100yahoo_quoted {margin-left:0 !important;border-left:1px #715FFA solid !important;padding-left:1ex !important;background-color:white;}Thank you so much!!



Sent from Yahoo Mail for iPhone


On Monday, February 27, 2017, 5:26 PM, kkprinceton <notifications@github.com> wrote:


I will aim for the end of the week. I have edited the code, but we need to test it out.

**Comment by kkprinceton** [comment]
We updated the package so that it now can accommodate any age when demo == TRUE. We made the changes in the development branch: https://github.com/kosukeimai/wru/tree/update-surname-handling. We are merging to the master branch now.

**Comment by standarddev** [comment]
blockquote, div.yahoo_quoted { margin-left: 0 !important; border-left:1px #715FFA solid !important; padding-left:1ex !important; background-color:white !important; } Great! I look forward to using it. 
Thanks again
Robin 


Sent from Yahoo Mail for iPhone


On Friday, March 3, 2017, 12:05 PM, kkprinceton <notifications@github.com> wrote:


We updated the package so that it now can accommodate any age when demo == TRUE. We made the changes in the development branch: https://github.com/kosukeimai/wru/tree/update-surname-handling. We are merging to the master branch now.
